scripts/battery3

```diff
@@ -51,7 +51,9 @@ if [ -z "$CHARGE_STATE" ]; then
     elif [ "$CHARGE_STATE" == "discharging" ]; then
     CHARGE_ICON=$(xrescat i3xrocks.label.dn D)
```

Comment:
If we're going away from the up = charge, down = discharge semantics, we'll need to replace discharging w/ something else to remain consistent.

Reply:
I agree. Right now the change will address the `charging,full-charged` status, and I can't think of any replacement label for `discharging`. So we can put this on hold until we get something that makes more sense than the current label.

Reply:
Do you mean to revert the change on line 54 or are you looking for a replacement for line 52?